src/goto-programs/goto_program_template.h

```diff
@@ -80,60 +79,129 @@ class goto_program_templatet
     return *this;
   }
 
-  /*! \brief Container for an instruction of the goto-program
-  */
+  /// This class represents a node in the control flow graph of the function.
```

Comment:
It's also that cfgt derives from grapht and hence allows the use of the graph algorithms (dominator analysis etc).